packages/protocol/contracts/Migrations.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.8;
+pragma solidity ^0.5.3;
```

Comment:
This was one of the changes suggested in the slither report, selected by @asaj as an issue that should be fixed. However, version-changes are rarely free of consequences. Is this a good idea?

Reply:
This was the message:
`Pragma version "^0.5.8" necessitates versions too recent to be trusted. Consider deploying with 0.5.3`